### PR TITLE
Locations

### PR DESCRIPTION
**Pull request by raminka13**
- [x] Added controllers and views to render locations.
- [x] Added some imgs from BSA.
- [x] Added routes for dynamic route from YML slug.

### CODE REVIEW

**Comment by raminka13** [comment]
This is far from perfect but a starting point to improve.

**Comment by timjedrek** [comment]
Thanks for working on it. I can't pull it up on my laptop just yet since I'm not home yet. Let's wait on merging to main until it's done. There's a lot of references to other clients.

File: app/views/locations/show.html.erb
Comment (timjedrek) [inline app/views/locations/show.html.erb:1]: image

They are not in St. Louis, they don't do helicopters, and they are not blue skies above (title and meta description)...

File: app/views/shared/_helicopter_airplane_fleet.html.erb
Comment (timjedrek) [inline app/views/shared/_helicopter_airplane_fleet.html.erb:1]: I'm assuming this aint being used...  This is legacy code from ideal.

File: app/views/shared/_cta_discovery_flight_left.html.erb
Comment (timjedrek) [inline app/views/shared/_cta_discovery_flight_left.html.erb:1]: Alt tag has st Louis arch...

File: app/views/shared/_phone_banner.html.erb
Comment (timjedrek) [inline app/views/shared/_phone_banner.html.erb:1]: I'm assuming this aint being used either.. wrong number as well.

File: config/routes.rb [hunk 1]
```diff
@@ -29,7 +30,7 @@
   resources :quiz_results
 
   # Enrollment form
-  #get 'enroll-at-simplifly', to: 'enrollments#new', as: :enroll
+  get 'enroll-at-suncity', to: 'enrollments#new', as: :enroll
```
Comment (timjedrek) [inline config/routes.rb:33]: I believe this is the popup modal form (enrollments) and there is no page which is why it was commented out

File: config/routes.rb [hunk 2]
```diff
@@ -115,9 +116,13 @@
   #get 'timeline-to-be-a-pilot', to: 'pages#timeline', as: :timeline
   #get 'pilot-training', to: 'pages#pilot_training', as: :pilot_training
 
-  #get 'pilot-programs', to: 'pages#pilot_programs', as: :programs
+  get 'pilot-programs', to: 'pages#pilot_programs', as: :programs
```
Comment (timjedrek) [inline config/routes.rb:119]: This is a St. Louis page....

File: config/routes.rb [hunk 3]
```diff
@@ -115,9 +116,13 @@
...
   get 'our-team', to: 'pages#our_team', as: :our_team
 
+  # Dynamic Location pages
+  get 'pilot-training', to: 'pages#pilot_programs'
```
Comment (timjedrek) [inline config/routes.rb:123]: This is a St. Louis page...

**Pull request merged**